### PR TITLE
added application.properties file for the addition of image creation properties

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**

### CODE REVIEW

File: pkg/java/v1/scaffolds/internal/templates/applicationproperties.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	return strings.ReplaceAll(s, ".", FilePathSeparator)
+}
+
+func (f *ApplicationPropertiesFile) SetTemplateDefaults() error {
+	if f.ProjectName == "" {
+		return fmt.Errorf("invalid Application Properties name")
+	}
+
+	if f.Path == "" {
+			f.Path = prependJavaPathResources("application.properties")
```
Comment (jmrodri) [inline pkg/java/v1/scaffolds/internal/templates/applicationproperties.go]: Looks like an extra tab here.

File: pkg/java/v1/scaffolds/internal/templates/applicationproperties.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+type ApplicationPropertiesFile struct {
+	file.TemplateMixin
+	OrgName         string
+	ProjectName     string
+}
+
+const (
+	FilePathSeparator = string(filepath.Separator)
+	javaPaths    = "src" + FilePathSeparator + "main" + FilePathSeparator + "resources"
```
Comment (jmrodri) [inline pkg/java/v1/scaffolds/internal/templates/applicationproperties.go]: there are a bunch of extra spaces between `javaPaths` and the `=`.

**Review by jmrodri — CHANGES_REQUESTED** [review]
2 small nits.

File: pkg/java/v1/scaffolds/internal/templates/applicationproperties.go (hunk 2 above)
Comment (laxmikantbpandhare) [reply pkg/java/v1/scaffolds/internal/templates/applicationproperties.go]: Modified according to suggestion

File: pkg/java/v1/scaffolds/internal/templates/applicationproperties.go (hunk 1 above)
Comment (laxmikantbpandhare) [reply pkg/java/v1/scaffolds/internal/templates/applicationproperties.go]: Modified according to suggestion

**Review by jmrodri — APPROVED** [review]

**Pull request merged**